c/driver/postgresql/postgres_copy_reader.h

```diff
@@ -1166,13 +1166,41 @@ class PostgresCopyBooleanFieldWriter : public PostgresCopyFieldWriter {
   }
 };
 
+template <typename T, T kOffset = 0>
```

Comment:
The offset will be used for things like timestamps, dates?

Reply:
Yea that is what it is used for on the reader side - I just copied the template over from that, but still have to implement the DATE support in another PR